### ISSUE TITLE
Typo mismatch

### ISSUE THREAD

**Issue by rsamanez** [ten-point-nine/freETarget#36]
Hi,
Actually, I'm not sure if this is a typo or the code was modified for some reason.
It's in the Arduino software, in the "freeETarget.ino" file, in the arm() implementation function, line 380
```
if ( sensor_status & TRIP_EAST )
```
I think it should be changed with
```
if ( sensor_status & TRIP_NORTH )
```

**Comment by ten-point-nine** [comment]
Thank you I’ll look into it.

Did it cause an error, or how did you find it?

Thanks

Allan

**Comment by ten-point-nine** [comment]
You are correct it is a typo.Fortunately it doesn’t affect the normal operation of the target.I’ll see if it was copied to Version 5ThanksAllanSent from my iPhoneOn Sep 24, 2024, at 13:06, Rommel Samanez ***@***.***> wrote:﻿
Hi,
Actually, I'm not sure if this is a typo or the code was modified for some reason.
It's in the Arduino software, in the "freeETarget.ino" file, in the arm() implementation function, line 380
if ( sensor_status & TRIP_EAST )

I think it should be changed with
if ( sensor_status & TRIP_NORTH )

**Comment by ten-point-nine** [comment]
ThanksI checked version 5 and that portion does the test a different way, so the bug was not propagated AllanSent from my iPhoneOn Sep 24, 2024, at 13:31, Allan Brown ***@***.***> wrote:﻿Thank you I’ll look into it.Did it cause an error, or how did you find it?ThanksAllanOn Sep 24, 2024, at 1:06 PM, Rommel Samanez ***@***.***> wrote:Hi,
Actually, I'm not sure if this is a typo or the code was modified for some reason.
It's in the Arduino software, in the "freeETarget.ino" file, in the arm() implementation function, line 380
if ( sensor_status & TRIP_EAST )
I think it should be changed with
if ( sensor_status & TRIP_NORTH )
